sdk/src/cameras/3d-smart-camera/camera_3d_smart.cpp

```diff
@@ -366,6 +378,7 @@ aditof::Status Camera3D_Smart::setFrameType(const std::string &frameType) {
         uint16_t afeRegsVal[2] = {0x0006, 0x0004};
         m_depthSensor->writeAfeRegisters(afeRegsAddr, afeRegsVal, 2);
         status = m_depthSensor->setFrameType(*frameDetailsIt);
+        status = m_rgbSensor->setFrameType(*frameDetailsIt);
```

Comment:
rgbSensor can't be set with the same frame type that is also used to set the depth sensor.

Reply:
This PR should focus on the RGBSensor class as described by the task and not on the Camera3DSmart class which is part of the high level API. I know that some stuff are missing from the camera class but that should be handled in a separate task/PR.